app/controllers/subscription/SubscriptionController.scala

```diff
@@ -37,12 +37,14 @@ class SubscriptionController @Inject()(logging: Logging,
                                       ) extends AuthenticatedController {
 
   def subscribe(nino: String): Action[AnyContent] = Action.async { implicit request =>
+    val path = request.headers.get(ITSASessionKeys.RequestURI).get //Will fail if request uri is not included in headers
```

Comment:
Maybe we should handle this more gracefully? 

I'm thinking we shouldn't throw a runtime error if the header doesn't include the path; instead, it would be good log it as an Error so that we can pick this up in Kibana.

How about something like:

```
    val noAuditPath = "-"
    val path = request.headers.get(ITSASessionKeys.RequestURI).fold {
      logging.err(s"Expected ${ITSASessionKeys.RequestURI} in HeaderCarrier, using '$noAuditPath' as audit path")
      noAuditPath
    }(requestUri => requestUri)
```
Or something similar?

At least that way, we can still get them subscribed and we can investigate why the RequestUri wasn't in the Header?